[BUG] cuDNN error when building profiler

**Describe the bug**

Compiling `cudnn_helpers.cpp.o` for the profiler fails due to namespace errors. Generally, I cannot find much documentation on the optional cuDNN dependency support; if you `grep` around `cutlass/media/docs/`, it is only briefly mentioned in the Quickstart and Profiler docs, mentioning that >= 7.6 is required.


**Steps/Code to reproduce bug**

```
git clone https://github.com/NVIDIA/cutlass
cd cutlass
export CUTLASS_PATH=...
export CUDNN_PATH=...
mkdir build && cd build
cmake .. -DCUTLASS_NVCC_ARCHS=80 -DCUTLASS_ENABLE_CUBLAS=ON -DCUTLASS_ENABLE_CUDNN=ON
make cutlass_profiler -j32
```

The CMake command completes without issue. It correctly identifies the cuDNN install location:
```
//Path to a file.
_CUDNN_INCLUDE_DIR:PATH=/soft/libraries/cudnn/cudnn-11-linux-x64-v8.6.0.163/include

//Path to a library.
_CUDNN_LIBRARY:FILEPATH=/soft/libraries/cudnn/cudnn-11-linux-x64-v8.6.0.163/lib/libcudnn.so
```

However, the `make` command fails right at the end:

```
[ 98%] Built target cutlass_library_symm_sm80_gz884hemm
Consolidate compiler generated dependencies of target cutlass_profiler
[ 98%] Building CXX object tools/profiler/CMakeFiles/cutlass_profiler.dir/src/cudnn_helpers.cpp.o
In file included from /soft/datascience/conda/2023-09-29/cutlass/tools/util/include/cutlass/util/device_memory.h:41,
                 from /soft/datascience/conda/2023-09-29/cutlass/tools/profiler/include/cutlass/profiler/cudnn_helpers.h:42,
                 from /soft/datascience/conda/2023-09-29/cutlass/tools/profiler/src/cudnn_helpers.cpp:38:
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:958:30: error: 'float_round_style' has not been declared in 'std'
  958 | using CUTLASS_STL_NAMESPACE::float_round_style;
      |                              ^~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:959:30: error: 'round_indeterminate' has not been declared in 'std'
  959 | using CUTLASS_STL_NAMESPACE::round_indeterminate;
      |                              ^~~~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:960:30: error: 'round_toward_zero' has not been declared in 'std'
  960 | using CUTLASS_STL_NAMESPACE::round_toward_zero;
      |                              ^~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:961:30: error: 'round_to_nearest' has not been declared in 'std'
  961 | using CUTLASS_STL_NAMESPACE::round_to_nearest;
      |                              ^~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:962:30: error: 'round_toward_infinity' has not been declared in 'std'
  962 | using CUTLASS_STL_NAMESPACE::round_toward_infinity;
      |                              ^~~~~~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:963:30: error: 'round_toward_neg_infinity' has not been declared in 'std'
  963 | using CUTLASS_STL_NAMESPACE::round_toward_neg_infinity;
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:966:30: error: 'float_denorm_style' has not been declared in 'std'
  966 | using CUTLASS_STL_NAMESPACE::float_denorm_style;
      |                              ^~~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:967:30: error: 'denorm_indeterminate' has not been declared in 'std'
  967 | using CUTLASS_STL_NAMESPACE::denorm_indeterminate;
      |                              ^~~~~~~~~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:968:30: error: 'denorm_absent' has not been declared in 'std'
  968 | using CUTLASS_STL_NAMESPACE::denorm_absent;
      |                              ^~~~~~~~~~~~~
/soft/datascience/conda/2023-09-29/cutlass/include/cutlass/platform/platform.h:969:30: error: 'denorm_present' has not been declared in 'std'
  969 | using CUTLASS_STL_NAMESPACE::denorm_present;
      |                              ^~~~~~~~~~~~~~
make[3]: *** [tools/profiler/CMakeFiles/cutlass_profiler.dir/build.make:202: tools/profiler/CMakeFiles/cutlass_profiler.dir/src/cudnn_helpers.cpp.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:23041: tools/profiler/CMakeFiles/cutlass_profiler.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:23048: tools/profiler/CMakeFiles/cutlass_profiler.dir/rule] Error 2
make: *** [Makefile:9833: cutlass_profiler] Error 2
```

**Expected behavior**

**Environment details (please complete the following information):**
- Linux SLES
- NVIDIA A100
- CUDA 11.8
- cuDNN v8.6.0.163

**Additional context**
Add any other context about the problem here.

--- a/include/cutlass/platform/platform.h
+++ b/include/cutlass/platform/platform.h
@@ -55,7 +55,7 @@
  *   (2) Re-implementations of STL functions and types:
  *       - C++ features that need the \p __device__ annotation.  These are
  *         placed into the \p platform namespace.
- *           - \p abs 
+ *           - \p abs
  *           - \p plus
  *           - \p less
  *           - \p greater
@@ -127,6 +127,7 @@
 #include <algorithm>   // Minimum/maximum operations
 #include <cstddef>     // nullptr_t
 #include <functional>  // Arithmetic operations
+#include <limits>      // float_round_style, float_denorm_style
 #include <utility>     // For methods on std::pair
 #if (!defined(_MSC_VER) && (__cplusplus >= 201103L)) || (defined(_MSC_VER) && (_MS_VER >= 1500))
 #include <type_traits>  // For integral constants, conditional metaprogramming, and type traits

